pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -1629,3 +1631,62 @@ func Test_isSchedulableAfterPodSchedulingContextChange(t *testing.T) {
 		})
 	}
 }
+
+func Test_isSchedulableAfterResourceSliceChange(t *testing.T) {
+	testcases := map[string]struct {
+		pod            *v1.Pod
+		claims         []*resourceapi.ResourceClaim
+		oldObj, newObj interface{}
+		expectedHint   framework.QueueingHint
+		expectedErr    bool
+	}{
+		"queue-new-resource-slice": {
+			pod:          podWithClaimName,
+			claims:       []*resourceapi.ResourceClaim{pendingClaim},
+			newObj:       resourceSlice,
+			expectedHint: framework.Queue,
+		},
+		"queue-update-resource-slice": {
+			pod:          podWithClaimName,
+			claims:       []*resourceapi.ResourceClaim{pendingClaim},
+			oldObj:       resourceSlice,
+			newObj:       resourceSliceUpdated,
+			expectedHint: framework.Queue,
+		},
+		"skip-not-find-resource-claim": {
+			pod:          podWithClaimName,
+			claims:       []*resourceapi.ResourceClaim{},
+			oldObj:       resourceSlice,
+			newObj:       resourceSlice,
+			expectedHint: framework.QueueSkip,
+		},
+		"skip-unexpected-object": {
+			pod:          podWithClaimName,
+			claims:       []*resourceapi.ResourceClaim{pendingClaim},
+			oldObj:       scheduling,
+			newObj:       scheduling,
+			expectedErr:  true,
+			expectedHint: framework.Queue,
+		},
+	}
+	for name, tc := range testcases {
+		tc := tc
+		t.Run(name, func(t *testing.T) {
+			t.Parallel()
+			logger, _ := ktesting.NewTestContext(t)
+			features := feature.Features{
+				EnableDynamicResourceAllocation: true,
+				EnableDRAControlPlaneController: true,
```

Comment:
nit: Since the resource slice is only used in Structured Parameters (`DynamicResourceAllocation`), I don’t see a reason to enable Classic DRA (`DRAControlPlaneController`) here.

Reply:
done